server/models/user.js

```diff
@@ -0,0 +1,25 @@
+import mongoose from 'mongoose';
+
+const { Schema } = mongoose;
+
+const UserSchema = new Schema({
+  name: {
+    type: String,
+  },
+  email: {
+    type: String,
+    unique: true,
+    required: true,
+  },
+  password: {
+    type: String,
+    required: true,
+  },
+  username: {
+    type: String,
+    unique: true,
+  },
+});
+
+const User = mongoose.model('User', UserSchema);
+module.exports = User;
```

Comment:
Newline missing at end of the file and here again use export default

Reply:
:+1: